src/saving/SaveHelper.cs

```diff
@@ -41,6 +41,27 @@ public enum SaveOrder
         FileSystem,
     }
 
+    /// <summary>
+    ///   The error that is returned after performing quick load.
+    /// </summary>
+    public enum QuickLoadError
+    {
+        /// <summary>
+        ///   No error, quick load is successful (or error is not handled).
+        /// </summary>
+        None,
+
+        /// <summary>
+        ///   The loaded version do not match the current game version.
+        /// </summary>
+        VersionMismatch,
+
+        /// <summary>
+        ///   Quick load is made not allowed to be performed.
```

Comment:
```suggestion
        ///   Quick load is currently prevented by the current state of the game.
```
I think this is slightly better wording, but not perfect either